watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/tests/test_posttreatment.py

```diff
@@ -72,24 +72,24 @@ def test_ideal_naocl_chlorination():
 def test_ideal_naocl_chlorination_full_block():
     model = run_chlorination_block_example(fix_free_chlorine=True)
     assert model.fs.ideal_naocl_mixer_unit.dosing_rate.value == pytest.approx(
-        1.7296113311683092e-09, rel=1e-3
+        9.505698559578499e-07, rel=1e-3
     )
     assert model.fs.ideal_naocl_mixer_unit.outlet.flow_mol[0].value == pytest.approx(
         25.000025535888078, rel=1e-3
     )
     assert model.fs.ideal_naocl_mixer_unit.outlet.mole_frac_comp[
         0, "OCl_-"
-    ].value == pytest.approx(1.6123004572288052e-07, rel=1e-3)
+    ].value == pytest.approx(5.107133802822922e-07, rel=1e-3)
 
     assert model.fs.ideal_naocl_chlorination_unit.free_chlorine.value == pytest.approx(
         2, rel=1e-3
     )
     assert model.fs.ideal_naocl_chlorination_unit.outlet.mole_frac_comp[
         0, "OCl_-"
-    ].value == pytest.approx(5.0027242332010015e-08, rel=1e-3)
+    ].value == pytest.approx(4.5088044031726496e-07, rel=1e-3)
     assert model.fs.ideal_naocl_chlorination_unit.outlet.mole_frac_comp[
         0, "H_+"
-    ].value == pytest.approx(1.49011416785194e-10, rel=1e-3)
+    ].value == pytest.approx(5.4260427865375997e-11, rel=1e-3)
```

Comment:
since these are all close to 0, wonder if they should have absolute tolerances instead of relative tolerances (what we typically want to use for ~0 values).

Reply:
Yea good catch, I think that should be the case.